admin/templates/version.h.template

```diff
@@ -0,0 +1,3 @@
+/* ${Herbert_AUTO_GENERATED_FILE_WARNING} */
+
+#define VERSION "${Herbert_SHORT_VERSION}"
```

Comment:
should this be `VERSION` or `HERBERT_VERSION` ? Just thinking we could have namespace issues; it's quite a common word.

Reply:
I've replaced this with a `constexpr` within a namespace. Using `constexpr` is better practice these days.